dask_cloudprovider/aws/ecs.py

```diff
@@ -1259,17 +1262,23 @@ async def _create_worker_task_definition_arn(self):
                         "resourceRequirements": resource_requirements,
                         "essential": True,
                         "command": [
-                            "dask-cuda-worker" if self._worker_gpu else "dask-worker",
-                            "--nthreads",
-                            "{}".format(
-                                max(int(self._worker_cpu / 1024), 1)
-                                if self._worker_nthreads is None
-                                else self._worker_nthreads
-                            ),
-                            "--memory-limit",
-                            "{}MB".format(int(self._worker_mem)),
-                            "--death-timeout",
-                            "60",
+                            e
+                            for e in [
+                                "dask",
+                                "cuda" if self._worker_gpu else None,
```

Comment:
Same here

Reply:
To both the above, just so it was more visible that we're filtering Nones out.
That seemed like the easiest way to have a variable number of arguments in the command.